packages/app/src/modules/metadata.js

```diff
@@ -1,6 +1,6 @@
 import i18n from '@dhis2/d2-i18n'
 
-export const relativePeriods = {
+const getRelativePeriods = () => ({
```

Comment:
Might be out of scope for this change, but since it has to do with i18n translated strings I thought it was relevant. Why do we keep a copy of all of these strings here when they could/should be fetched from https://github.com/dhis2/analytics/blob/da9b9d1fba9a0c62532293e2501863e61d33ecdf/src/components/PeriodDimension/utils/relativePeriods.js#L14 ?

Reply:
@janhenrikoverland

Reply:
@martinkrulltott Now that you added them to analytics they can be removed from the app, but the app must be updated, so sounds like a separate task.

Reply:
New ticket: https://jira.dhis2.org/browse/TECH-384 @janhenrikoverland @jenniferarnesen

Reply:
I'll work on it separately.